.gitignore

```diff
@@ -48,6 +48,7 @@ web/sites/simpletest
 private
 .env
 .env-template
+.lando.yml
```

Comment:
.lando.yml file should not be listed in .gitignore. This file is used by Lando to start up the server and needs to be the same among all developers. Please delete this line!